scripts/check_all_metrics.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
We could move the script into scripts/analyze_training_metrics.py

Reply:
In order to do that, it would be better if we moved the content of the `retrieve_training_metrics` in bugbug itself (something like `bugbug/metrics.py`) so we won't have to mangle with the Python path, would you agree?

Reply:
Yeah, but I'd like to keep the Taskcluster dependency out of bugbug core code.
Maybe we can have a TaskIndex base class, a TaskclusterTaskIndex extending the base class, and allow users to use other services if they want (implementing another extension of TaskIndex).
Anyway, we can keep the script for now and file an issue about this.